src/marquee/Marquee.style.tsx

```diff
@@ -15,6 +17,7 @@ export const StyledMarquee = styled.ul`
 export const StyledMarqueeItem = styled.li<{
   position: number
   totalItems: number
+  hasHorizontalSpacing?: boolean
 }>`
```

Comment:
`& NormalizeProps` instead of repeating it

Reply:
Thanks, still some getting used to manipulating types on my side.
I changed it, looks cleaner now.